pkg/datamodels/fiware/indoorenvironmentobserved.go

```diff
@@ -0,0 +1,25 @@
+package fiware
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/diwise/context-broker/pkg/ngsild/types"
+	"github.com/diwise/context-broker/pkg/ngsild/types/entities"
+	ed "github.com/diwise/context-broker/pkg/ngsild/types/entities/decorators"
+)
+
+func NewIndoorEnvironmentObserved(id, dateObserved string, decorators ...entities.EntityDecoratorFunc) (types.Entity, error) {
```

Comment:
På WeatherObserved tar vi in lat och lon och lägger på Location så. 
Location är required för IndoorEnv. Jag tycker såna ska tvingas in om vi gör en New.